velox/expression/tests/SparkExpressionFuzzerTest.cpp

```diff
@@ -44,7 +44,8 @@ DEFINE_string(
     "Supported special forms: and, or, coalesce, if, switch, cast.");
 
 int main(int argc, char** argv) {
-  facebook::velox::functions::sparksql::registerFunctions("");
+  const std::string prefix = "spark_";
```

Comment:
Do we still need this change?

Reply:
I think yes. I change function name in UT, but can not find a better way to solve keyword conflicts in fuzzer test, if don't add prefix.

Reply:
Can you not modify the fuzzer to wrap function names in double quotes?

Reply:
you mean modify the fuzzer framework to wrap all function names with double quotes to avoid the keyword problem when we add other functions?

Reply:
That's what I'm thinking about. It would be nice to make this change in a separate PR though.

Reply:
After read related code, I found expression fuzzer does not use DuckDB to parse expr, only aggregate fuzzer use DuckDB.
And I remove above changes, all CI passed.

Do you think we still need modify aggregate fuzzer to avoid potential keyword conflict problem?

Reply:
@Yohahaha Thank you for the follow up.

> Do you think we still need modify aggregate fuzzer to avoid potential keyword conflict problem?

We can do this, but looks like it is not necessary right now.